src/amber/cli/templates/scaffold/controller/granite/spec/controllers/{{name}}_controller_spec.cr.ecr

```diff
@@ -1,71 +1,86 @@
 <% field_name = @fields.first.name -%>
```

Comment:
Are the controllers and the tests any different between granite and crecto? Is there a way to structure these files so we don't have the same tests all written twice?